src/store/reducers/login-reducer.js

```diff
@@ -0,0 +1,23 @@
+const initialState = {
+    email: '',
+    isAuth: false,
+    password: ''
+};
+
+const loginReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'SET_USER_DATA':
+            return {
+                ...state,
+                email: action.email,
+                isAuth: true,
+                password: action.password
+            };
+        default:
+            return state;
+    }
+};
+
+export const setAuthUserData = (email, password) => ({ type: 'SET_USER_DATA', email, password });
```

Comment:
move actionTypes to the actionTypes.js.

Also move action to the src/store/actions/loginActions.js (use counterActions.js as an example)

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/store/actions/actionTypes.js#L1-L9